src/scrapers/leumi-card.js

```diff
@@ -151,10 +150,62 @@ async function getAccountNumber(page, cardIndex) {
   return accountNumber;
 }
 
+async function getChargedCurrencySymbolOfSection(page, cardSection) {
+  const chargedCurrencyElement = await cardSection.$('tbody:first-child > tr:first-child > th:nth-child(7) > a');
+
+  const currencyHeaderValue = await page.evaluate((a) => {
+    return a.innerText;
+  }, chargedCurrencyElement);
+  return currencyHeaderValue.charAt(10);
+}
+
+function parseAmount(amountStr) {
+  if (typeof amountStr === 'number') {
+    return amountStr;
+  }
+
+  if (typeof amountStr === 'undefined' || amountStr === null ||
+    (typeof amountStr === 'string' && amountStr.trim().length === 0)) {
+    return null;
+  }
+
+  const formattedAmount = amountStr
+    .replace(SHEKEL_CURRENCY_SYMBOL, '')
+    .replace(DOLLAR_CURRENCY_SYMBOL, '')
+    .replace(',', '')
+    .replace(/[ ]{2,}/g, ' ')
```

Comment:
@esakal maybe add a comment for dummies explaining what this line does. I still don't understand :-)